config/migrations/20230719135250-add-secretaris-mandatees.ttl

```diff
@@ -0,0 +1,62 @@
+@prefix mandaat:	<http://data.vlaanderen.be/ns/mandaat#> .
+@prefix org:	<http://www.w3.org/ns/org#> .
+@prefix mu:	<http://mu.semte.ch/vocabularies/core/> .
+@prefix dct:	<http://purl.org/dc/terms/> .
+@prefix person:	<http://www.w3.org/ns/person#> .
+@prefix persoon:	<https://data.vlaanderen.be/ns/persoon#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+<http://themis.vlaanderen.be/id/persoon/3571cad5-15c8-4589-a0d8-2de1b6695b51>	a person:Person ;
+	persoon:gebruikteVoornaam	"Jeroen" ;
+	foaf:familyName	"Overmeer" ;
+	mu:uuid	"3571cad5-15c8-4589-a0d8-2de1b6695b51" .
+
+<http://themis.vlaanderen.be/id/persoon/1d4c8917-edc7-495f-a202-55511f459236>	a person:Person ;
+	persoon:gebruikteVoornaam	"Gert" ;
+	foaf:familyName	"Eeraerts" ;
+	mu:uuid	"1d4c8917-edc7-495f-a202-55511f459236" .
+
+<http://themis.vlaanderen.be/id/persoon/1f5486c1-ac11-43e6-a608-7289afad0ad0>	a person:Person ;
+	persoon:gebruikteVoornaam	"Dries" ;
+	foaf:familyName	"Verhaeghe" ;
+	mu:uuid	"1f5486c1-ac11-43e6-a608-7289afad0ad0" .
+
+<http://themis.vlaanderen.be/id/persoon/467ecb38-d8b5-4d41-905b-5c58966dbc86>	a person:Person ;
+	persoon:gebruikteVoornaam	"Raf" ;
+	foaf:familyName	"Suys" ;
+	mu:uuid	"467ecb38-d8b5-4d41-905b-5c58966dbc86" .
+
+<http://themis.vlaanderen.be/id/mandataris/f57a9d61-84ce-438e-b601-a905704ed50d> a mandaat:Mandataris ;
+	mandaat:isBestuurlijkeAliasVan	<http://themis.vlaanderen.be/id/persoon/3571cad5-15c8-4589-a0d8-2de1b6695b51> ;
+	mandaat:start	"2019-10-02T00:00:00Z"^^xsd:dateTime ;
```

Comment:
kloppen deze starttijden? Moeten deze gelijk zijn met de huidige ministers (dus sinds aantreden brouns 18/05/2022)?

Reply:
Ik zou zeggen dat we ze zo laten en als we horen dat het anders moet kunnen we later extra mandaten toevoegen in het verleden. Voor huidige agenda's zal dat geen probleem zijn (we beschouwen de huidige mandatarissen als de meest recente, dan staat de link al juist) en dan hoeven we enkel te backfillen met secretarissen (die nergens aan gelinkt zijn).

Reply:
voor mij ok. Denk niet dat wie datums ergens tonen voor deze.